src/components/NumberInput/NumberInput.scss

```diff
@@ -0,0 +1,30 @@
+@use '../../styles/scss/typography.scss';
+@use '../../styles/scss/paddings.scss';
+@use '../../styles/scss/margins.scss';
+@use '../../styles/scss/colors.scss';
+@use '../../styles/scss/mixins.scss';
+@use '../../styles/scss/variables.module.scss';
+
+.#{variables.$prefix}-number-input {
+  display: flex;
+  flex-direction: column;
+  padding: paddings.$xs;
+  margin: margins.$xs;
+  color: colors.$text;
+
+  @include typography.text-regular();
+
+  label {
+    margin-bottom: margins.$xs;
+  }
+
+  input {
+    width: 100%;
+    min-width: 200px;
+    max-width: 500px;
+    padding: 1px paddings.$xs;
```

Comment:
Create a variable in `paddings.scss` for input-inner-padding with the value of `1px`

Reply:
Refactored in pair programming session